src/rosdoc_lite/doxygenator.py

```diff
@@ -194,7 +198,8 @@ def package_doxygen_template(template, rd_config, path, package, html_dir, heade
               '$HTML_HEADER': header_filename,
               '$HTML_OUTPUT': os.path.realpath(html_dir),
               '$IMAGE_PATH': rd_config.get('image_path', path), #default to $INPUT
-              '$INPUT':  path, '$PROJECT_NAME': package,
+              '$INPUT': ' '.join([path, mdfile]),
```

Comment:
It looks like this is going to leave trailing whitespace. Have you verified that doesn't create any warnings etc?

Reply:
The trailing space doesn't cause any trouble and/or warnings.